.gitignore

```diff
@@ -57,8 +57,8 @@ typings/
 # dotenv environment variables file
 .env
 
-# webpack bundle
-public/*.bundle.js*
-public/*.html
+# Ignore everything in the root of the public folder
```

Comment:
🐾 🔫 Um, surely there is a better way to do this where we wont be shooting ourselves in the foot later? Why not have assets also be copped from a src directory or have all the output go to `dist` and copy over anything in public?